Makefile

```
@@ -243,6 +247,7 @@ bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metada
         < config/manifests/bases/authorino-operator.clusterserviceversion.template.yaml \
         > config/manifests/bases/authorino-operator.clusterserviceversion.yaml
 	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle -q --overwrite --version $(BUNDLE_VERSION) $(BUNDLE_METADATA_OPTS) --package authorino-operator
+	($(YQ) e -e '.config.replaces' $(BUILD_CONFIG_FILE) && V="$(shell $(YQ) e -e '.config.replaces' $(BUILD_CONFIG_FILE))" $(YQ) eval '.spec.replaces = strenv(V)' -i $(BUNDLE_CSV)) || echo "no replaces added"
```

```suggestion
	($(YQ) e -e '.config.replaces' $(BUILD_CONFIG_FILE) && V="$(shell $(YQ) e -e '.config.replaces' $(BUILD_CONFIG_FILE))" $(YQ) eval '.spec.replaces = strenv(V)' -i bundle/manifests/authorino-operator.clusterserviceversion.yaml) || ($(YQ) eval '.' -i bundle/manifests/authorino-operator.clusterserviceversion.yaml && echo "no replaces added")
```

So we make sure it's equally indented by default and therefore comparing the diff to `main` can be focused on what matters.

---

Yep you're right we should just update the file indentation once and from then on it'll be clearer what's going on in each new release branch.